extensions/data-workspace/src/dialogs/openExistingDialog.ts

```diff
@@ -26,17 +24,7 @@ export class OpenExistingDialog extends DialogBase {
 	public locationRadioButtonFormComponent: azdataType.FormComponent | undefined;
 	public formBuilder: azdataType.FormBuilder | undefined;
 
-	private _targetTypes = [
-		{
-			name: constants.Project,
-			icon: this.extensionContext.asAbsolutePath('images/Open_existing_Project.svg')
-		}, {
-			name: constants.Workspace,
```

Comment:
should we keep workspace as an option in this dialog if people want to open multiple projects that are already in a workspace? Or we expect people to use the open workspace in the file menu now?

Reply:
looking at the telemetry, opening a workspace through this dialog doesn't have more than 10 users per day, so it might not be missed that much